platforms/php/versionsToBuild.txt

```diff
@@ -19,6 +19,7 @@
 8.0.3, c9816aa9745a9695672951eaff3a35ca5eddcb9cacf87a4f04b9fb1169010251, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
 8.0.11, e3e5f764ae57b31eb65244a45512f0b22d7bef05f2052b23989c053901552e16, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
 8.0.13, cd976805ec2e9198417651027dfe16854ba2c2c388151ab9d4d268513d52ed52, 1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F,
+8.0.16, f27a2f25259e8c51e42dfd74e24a546ee521438ad7d9f6c6e794aa91f38bab0a, 1729F83938DA44E27BA0F4D3DBDB397470D12172, BFDDD28642824F8118EF77909B67A5C12229118F,
```

Comment:
Can we add dynamic installation test for php new version?

Reply:
Updated. Thanks!